xigt/scripts/xigt_export.py

```diff
@@ -32,9 +33,11 @@ def main(arglist=None):
         help='The format of the output corpus (default: latex).')
     parser.add_argument('-c', '--config', metavar='PATH', default=None,
         help='A JSON-formatted configuration file.')
+    parser.add_argument('-r', '--relations', metavar='PATH', default=None,
+        help='A Relations file needed for converting to itsdb format.')
```

Comment:
Adding the `--relations` option on the `xigt export` command is not ideal because it is only relevant when `--format=itsdb` is used and not for other export formats. This is not a blocking issue, however.